### DISCUSSION TITLE
从 ASAN Stuck 到 Open Files Limit

### DISCUSSION THREAD

**Discussion by giscus** [zu1k/blog#70] [Comments]
# 从 ASAN Stuck 到 Open Files Limit

用了两天时间来定位一个 AddressSanitizer 卡住的问题，最终确定是由 Docker 最大文件打开数过大导致的，尝试修改 LLVM 源码来进行优化

https://zu1k.com/posts/linux/large-nofile-cause-asan-stuck/

**Comment by AaronYang238** [comment]
棒
